packages/react-server-dom-parcel/src/client/ReactFlightDOMClientBrowser.js

```diff
@@ -104,6 +125,7 @@ function startReadingFromStream(
 }
 
 export type Options = {
+  debugChannel?: {writable?: WritableStream, ...},
```

Comment:
@devongovett You might want to remove these options and instead just wire it up automatically in development to any HMR sockets in Parcel automatically on both ends.